eap6-support/hawkular-wildfly-agent-feature-pack-eap6/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -622,11 +623,11 @@
         <operation-dmr name="Shutdown" internal-name="shutdown" > <!-- shutdown and restart -->
```

Comment:
I don't know either, @pilhuhn tell us :]

I see your logic, those operations don't modify anything. The question is whether we want them enabled in the containers. According to that pet & cattle analogy, the containers should be shot in the leg = not restarting the processes in it.